modules/ROOT/pages/import-export-an-organization.adoc

```diff
@@ -13,28 +13,30 @@ image:images/UI2021.1/install-export.png[Install export organization]
 ====
 
 This will import a file containing your whole organization data. This organization data will be merged with existing data. +
-In case of conflict, the priority is given to the data in the imported file. Take care not to overwrite information that has been updated in Bonita Portal since the last installation of the organization.
+In case of conflict, the priority is given to the data in the imported file. Take care not to overwrite information that has been updated on the Bonita platform since the last installation of the organization.
 ====
 
 [NOTE]
 ====
 
-In 6.3 there was a change to the structure of the .xml file. This means that you cannot install into Bonita Portal 6.3.0 or later an organization .xml file that was created in 6.2.x or earlier. +
+In 6.3 there was a change to the structure of the .xml file. This means that you cannot install into Bonita platform 6.3.0 or later an organization .xml file that was created in 6.2.x or earlier. +
 You will need to first install the organization file into Bonita Studio and re-export it, so it is compatible.
```

Comment:
so it is compatible. => to make it compatible. or "for it to be compatible"

Reply:
Done